addons/portal/static/src/interactions/portal_composer.js

```diff
@@ -36,15 +36,18 @@ export class PortalComposer extends Interaction {
                 options[name] = parseInt(options[name]);
             }
         }
-        return Object.assign({
```

Comment:
(Sorry for diff but it looks like someone didn't activate their pre-commit, I'll post a comment on the markup change)